### PR TITLE
update auth configuration docs, don't expose oidc secret in api

### PR DESCRIPTION
**Pull request by opqdonut**
~on top of #1843 for practical reasons~ rebased

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -11,11 +11,21 @@ The full list of available configuration options can be seen in [config-defaults
...
+When using this option, login requests are directed to `/Shibboleth.sso/Login`. A link to an alternative login endpoint can also be shown if the `:alternative-login-url` option is set.
+
+### OpenId Connect (`:oidc`)
+
+The `:oidc` authentication method has the following configuration options:
+
+* `:oidc-domain` – the openid connect configration is fetched from `https://{oidc-domain}/.well-known/openid-configuration`
+* `:oidc-client-id`
+* `:oidc-client-secret`
+* `:public-url` - the redirect uri sent to the openid endpoint is `{:public-url}/oidc-callback`
```
Comment (luontola) [inline docs/configuration.md]: For consistency remove or add ":" in both `{oidc-domain}` and `{:public-url}`

File: src/clj/rems/api/public.clj [hunk 2]
```diff
@@ -61,4 +61,6 @@
       (ok (assoc env
                  :authentication "HIDDEN"
                  :database-url "HIDDEN"
-                 :test-database-url "HIDDEN")))))
+                 :test-database-url "HIDDEN"
+                 :oidc-client-id "HIDDEN"
```
Comment (luontola) [inline src/clj/rems/api/public.clj]: The client ID is public information, so it doesn't need to be hidden. It's present in the login redirect.

Comment (opqdonut) [reply src/clj/rems/api/public.clj]: right! fixed

File: docs/configuration.md (hunk 1 above)
Comment (opqdonut) [reply docs/configuration.md]: fixed

**Pull request merged**